awscli/clidriver.py

```diff
@@ -363,8 +363,10 @@ def _get_command_table(self):
 
     def _get_service_model(self):
         if self._service_model is None:
+            api_version = self.session.get_config_variable('api_versions').get(
```

Comment:
The default is an empty dictionary: https://github.com/boto/botocore/blob/develop/botocore/session.py#L83. So we can assume that value will always be a dictionary unless they messed up the way in which they specified the value in the config file like: ``api_versions = foo`` which will cause it to be a string.